src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+const ADD = 'ADD_BOOK';
+const REMOVE = 'REMOVE_BOOK';
+
```

Comment:
- [ ] In keeping the ducks pattern kindly refactor these action types.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK'
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK'
  ```

  You read [this article for more](https://medium.com/@matthew.holman/what-is-redux-ducks-46bcb1ad04b7)